projects/packages/my-jetpack/_inc/components/connection-status-card/styles.module.scss

```diff
@@ -84,6 +87,45 @@ $myjetpack-connection-avatar-overlap: 10px;
 	}
 }
 
+.connection-state {
+
+	h4 {
+		margin-top: 0;
+		margin-bottom: 0.25rem;
+
+		:global(button.components-button) {
+			height: auto;
```

Comment:
Would we need those styles if we didn't swap the Button component source package?

Reply:
Yes, and we now prefer using Core components whenever possible.